pkg/manifests/manifests.go

```diff
@@ -451,7 +454,9 @@ func (f *Factory) AlertmanagerMain(host string, trustedCABundleCM *v1.ConfigMap)
 
 	a.Spec.Image = &f.config.Images.Alertmanager
 
-	a.Spec.ExternalURL = f.AlertmanagerExternalURL(host).String()
+	if f.consoleConfig != nil {
+		a.Spec.ExternalURL = f.consoleConfig.Status.ConsoleURL
```

Comment:
Sorry I missed this last review: we want to append the path component `/monitoring`. See https://github.com/openshift/cluster-monitoring-operator/pull/1532#discussion_r785567207